### PR TITLE
Added "Save OPcode" Button to spanish translation in intro

### PR DESCRIPTION
**Pull request by sebastianbarry**
login-es.html
- Added the Save button to the Spanish translation

### CODE REVIEW

**Comment by sebastianbarry** [comment]
This is what the change looks like

Screen Shot 2022-07-12 at 10 19 20 AM

I noticed these following areas in the Spanish version that are not translated:

Screen Shot 2022-07-12 at 10 18 53 AM

Screen Shot 2022-07-12 at 11 24 34 AM

It doesn't look like these are changes that can be made in e-mission-translate, rather changes in e-mission-phone. But I am unsure of how to translate in the [e-mission-phone/www/js/intro.js](https://github.com/e-mission/e-mission-phone/blob/master/www/js/intro.js) 

Here is an example of English being hard-coded
```
 $scope.typeExisting = function() {
    $scope.data = {};
    const tokenPopup = $ionicPopup.show({
        template: '<input type="String" ng-model="data.existing_token">',
        title: 'Enter the existing token that you have',
        scope: $scope,
        buttons: [
          {
            text: '<b>OK</b>',
            type: 'button-positive',
            onTap: function(e) {
              if (!$scope.data.existing_token) {
                //don't allow the user to close unless he enters a username

                e.preventDefault();
              } else {
                return $scope.data.existing_token;
              }
            }
          },{
            text: '<b>Cancel</b>',
            type: 'button-stable',
            onTap: function(e) {
              return null;
            }
          }
        ]
    });
```

Here is what I added with hard-coded english in [my branch](https://github.com/sebastianbarry/e-mission-phone/blob/save-opcode-at-login/www/js/intro.js)

```
  $scope.saveLogin = function(token) {
    if(token == "") {
      token = $scope.randomToken;
    }
    $scope.currentToken = token;
    
    const savePopup = $ionicPopup.show({
      template: '<center><qrcode class="col" aria-label="qrcode for user email" data="{{currentToken}}" size="220" download></qrcode></center>'
              + '<div class="col" style="height: 150%"><button class="col" style="height: 100%; background-color: #01D0A7" ng-click="shareQR()" class="control-icon-button"> <u>{{currentToken}}</u> </button></div>',
      title: 'Make sure to save your OPcode!<br>',
      scope: $scope,
      buttons: [
        {
          text: '<b>Continue</b>',
          type: 'button-positive',
          onTap: function(e) {
            $scope.alreadySaved = true;
          }
        }
      ]
    })
  };
```

I see that there is `$translate` in the intro.js code, but I don't see any examples of it being used in intro.js
```
$ionicPlatform.ready().then(() => {
      DynamicConfig.configReady().then((newConfig) => {
        Logger.log("Resolved UI_CONFIG_READY promise in intro.js, filling in templates");
        $scope.lang = $translate.use();
        $scope.ui_config = newConfig;
        // TODO: we should be able to use $translate for this, right?
        $scope.template_text = newConfig.intro.translated_text[$scope.lang];
        if (!$scope.template_text) {
            $scope.template_text = newConfig.intro.translated_text["en"]
        }
        $scope.init();
      });
    });
```

**Comment by shankari** [comment]
> I noticed these following areas in the Spanish version that are not translated:

woops! 

If you search for `$translate` in the rest of the js code, we get

```
$ grep -r translate www/js/
```

has multiple results like `$translate.instant('general-settings.nuke-ui-state-only')`

File: es/i18n/intro/login-es.html [hunk 1]
```diff
@@ -2,6 +2,10 @@
 <div class="intro-title">
 <center><h3>Iniciar sesión mediante token anónimo (OPcode)</h3></center>
 
+<div class="save-button">
+  <center><div class="col" ng-click="saveLogin('{{randomToken}}')" class="control-icon-button"> <br>Guarde su OPcode <i class="ion-ios-download-outline" style="font-size: 200%"></i></div></center>
+</div>
+
```
Comment (shankari) [inline es/i18n/intro/login-es.html:8]: This should be below line 10.
Note that this file and the file that you are trying to check in to `e-mission-phone` are subtly different
https://github.com/e-mission/e-mission-phone/pull/863/files#diff-0e3ef9886230bc415ac090d1f91f42cd04679f2059358b0ac87e453218b24f39R6
and both are wrong 😦 

This has the save button above the OPcode, and the other one does not have the `program_or_study=='study' fix.
This is why it is important to not check in two copies of the file, and to ensure the version that you are committing is correct.

**Review by shankari — CHANGES_REQUESTED** [review]
Otherwise looks good.

**Comment by sebastianbarry** [comment]
I moved the Save randomToken for the OPcode from the Spanish version of the 'program' version of the app:
Screen Shot 2022-07-13 at 5 36 52 PM

**Review by shankari — APPROVED** [review]

**Pull request merged**